<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTimeDialog</class>
 <widget class="QDialog" name="QTimeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTimeDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="title">
    <string>Time Info</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>StartDate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>EndDate</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="startDateTimeEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Batang&quot;;</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="endDateTimeEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Batang&quot;;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>239</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Bell MT&quot;;</string>
   </property>
   <property name="text">
    <string>Comfirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>239</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Bell MT&quot;;</string>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
